<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>application_pages</class>
 <widget class="QStackedWidget" name="application_pages">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StackedWidget</string>
  </property>
  <property name="currentIndex">
   <number>3</number>
  </property>
  <widget class="QWidget" name="page_1">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QFrame" name="frame_Principal">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>280</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>280</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0">
         <item row="2" column="1">
          <widget class="QPushButton" name="btnCarregarPDF">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	background-color: rgb(67, 133, 200);
	border: 2px solid #c3ccdf;
	color: rgb(255, 255, 255);
	border-radius: 10px;
}
QPushButton:hover{
	background-color: rgb(85, 170, 255);
}
QPushButton:pressed{
	background-color: rgb(255, 0, 127)	
}</string>
           </property>
           <property name="text">
            <string>Carregar</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 700 14pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Caminho Salvar</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0" colspan="2">
          <widget class="QPushButton" name="btnSalvarPDF">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	background-color: rgb(67, 133, 200);
	border: 2px solid #c3ccdf;
	color: rgb(255, 255, 255);
	border-radius: 10px;
}
QPushButton:hover{
	background-color: rgb(85, 170, 255);
}
QPushButton:pressed{
	background-color: rgb(255, 0, 127)	
}</string>
           </property>
           <property name="text">
            <string>Converter PDF para PNG</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLineEdit" name="edtCaminhoPDF">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit{
	background-color: rgb(68, 71, 90);
	padding: 8px;
	border: 2px solid #c3ccdf;
	color: rgb(255, 255, 255);
	border-radius: 10px;
}</string>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="btnCarregarCaminhoSalvarPDF">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	background-color: rgb(67, 133, 200);
	border: 2px solid #c3ccdf;
	color: rgb(255, 255, 255);
	border-radius: 10px;
}
QPushButton:hover{
	background-color: rgb(85, 170, 255);
}
QPushButton:pressed{
	background-color: rgb(255, 0, 127)	
}</string>
           </property>
           <property name="text">
            <string>Carregar</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLineEdit" name="edtCaminhoSalvarPDF">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit{
	background-color: rgb(68, 71, 90);
	padding: 8px;
	border: 2px solid #c3ccdf;
	color: rgb(255, 255, 255);
	border-radius: 10px;
}</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_ola">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 700 14pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Caminho Arquivo PDF</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QCheckBox" name="checkBox">
           <property name="text">
            <string>Abrir pasta após conversão?</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="lbl_msg_conversao">
         <property name="styleSheet">
          <string notr="true">QLabel{
	padding: 2px;
	color: #F7122D
}</string>
         </property>
         <property name="text">
          <string>lbl_msg_conversao</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="page_2">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QFrame" name="frame_toggle_button">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>250</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_page_2">
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="spacing">
            <number>10</number>
           </property>
           <item row="0" column="2">
            <widget class="QSlider" name="horizontalSlider">
             <property name="minimum">
              <number>60</number>
             </property>
             <property name="maximum">
              <number>300</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="0" alignment="Qt::AlignLeft">
            <widget class="QLabel" name="label_2">
             <property name="styleSheet">
              <string notr="true">font: 700 14pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>Animação</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" alignment="Qt::AlignLeft">
            <widget class="QLabel" name="label_4">
             <property name="styleSheet">
              <string notr="true">font: 700 14pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>Largura</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QComboBox" name="comboBox">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QComboBox{
	background-color: rgb(67, 133, 200);
	border: 2px solid #c3ccdf;
	color: rgb(255, 255, 255);
	border-radius: 10px;
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QCheckBox" name="chk_exemplo">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="text">
            <string>CheckBox</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="page_circular_progress_bar">
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QFrame" name="frame_circular_progress_bar">
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>700</width>
        <height>300</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QGridLayout" name="grid_layout_layout_circular_progress_bar"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="page_3">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>150</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>150</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>14</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 700 14pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Tema</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxTema">
           <property name="minimumSize">
            <size>
             <width>350</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>350</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox{
	background-color: rgb(67, 133, 200);
	border: 2px solid #c3ccdf;
	color: rgb(255, 255, 255);
	border-radius: 10px;
}</string>
           </property>
           <property name="currentText">
            <string/>
           </property>
           <property name="insertPolicy">
            <enum>QComboBox::InsertAtBottom</enum>
           </property>
           <property name="placeholderText">
            <string>teste</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pshAplicarConfiguracoes">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	background-color: rgb(67, 133, 200);
	border: 2px solid #c3ccdf;
	color: rgb(255, 255, 255);
	border-radius: 10px;
}
QPushButton:hover{
	background-color: rgb(85, 170, 255);
}
QPushButton:pressed{
	background-color: rgb(255, 0, 127)	
}</string>
           </property>
           <property name="text">
            <string>Aplicar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
